docs/connectors/mysql.asciidoc

```diff
@@ -978,18 +978,36 @@ However, when `decimal.handling.mode` configuration property is set to `double`,
 |Semantic type (schema name)
 |Notes
 
-|`NUMERIC[(M[,D])]` 
+|`NUMERIC[(M[,D])]`
 |`FLOAT64`
 |
 |
 
-|`DECIMAL[(M[,D])]` 
+|`DECIMAL[(M[,D])]`
 |`FLOAT64`
 |
 |
 
 |=======================
 
+[[spatial-types]]
+==== Spatial Data Types
+
+The MySQL connector also has limited support for some of the following https://dev.mysql.com/doc/refman/5.7/en/spatial-datatypes.html[spatial data types]:
```

Comment:
Actually, maybe it be better to say "As of version 0.5.1, the MySQL connector ..."